packages/ssi-service/src/auth.js

```diff
@@ -1,46 +1,43 @@
-import Jwt, { SsoEnv } from 'jwt-redhat';
+import Keycloak from 'keycloak-js';
 
 /**
  * Helper class for Red Hat's Internal SSO Authentication
  */
 class OpAuth {
   constructor () {
-    this._jwt = Jwt;
-
-    this._jwtOptions = {
-      keycloakOptions: {
-        realm: 'EmployeeIDP',
-        clientId: 'one-portal',
-        internalAuth: true,
-      },
-      keycloakInitOptions: {
-        responseMode: 'fragment',
-        flow: 'implicit',
-      },
-      ssoEnv: process.env.NODE_ENV === 'production' ? SsoEnv.PROD : SsoEnv.STAGE,
+    this.keycloakOptions = {
+      clientId: 'one-portal',
+      realm: 'EmployeeIDP',
+      url: process.env.NODE_ENV === 'production' ? 'https://auth.redhat.com/auth' : 'https://auth.stage.redhat.com/auth',
```

Comment:
Can you please use the environment variables for this URLs

Reply:
It's already using environment variable for `NODE_ENV`.

I don't see a point in storing the SSO urls in environment variables since they're constant.

Reply:
I meant the internal auth urls which you statically used here ```'https://auth.redhat.com/auth' : 'https://auth.stage.redhat.com/auth',
```

Reply:
Changed it to an env variable